### PR TITLE
Add blog to gcmp

### PR DESCRIPTION
**Pull request by trishalim**
Don't merge because I need to remove the test blog post

Test here https://gcmp-homepage-git-trishalim-jazz-419-gcmp.vercel.app/news

Typography (prose) needs some work, but I'll do that separately.

### CODE REVIEW

**Comment by aeplay** [comment]
Looks great already!

How about we use MDX exports (https://mdxjs.com/guides/frontmatter/) as a front matter-style way to have both post content and metadata in one file?

And then the manually populated array of all imported posts is still a good way to do it until we come up with something more clever

File: homepage/gcmp/components/blog/posts/test.mdx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  title: "Test",
+  date: "2023-01-01",
+  coverImage: "/social-image.png",
+  author: {
+    name: "Anselm Eickhoff",
+    picture: "/social-image.png",
+  },
+  excerpt:
+    "Jazz is a framework for local-first data/permissions. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut gravida vel urna sit amet lacinia. Morbi euismod mi ac lacus feugiat, vel sollicitudin urna faucibus. ",
+};
```
Comment (trishalim) [inline homepage/gcmp/components/blog/posts/test.mdx:12]: @aeplay meta is now here

File: homepage/gcmp/lib/posts.tsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import * as TestPost from "@/components/blog/posts/test.mdx";
+
+export const posts = [TestPost];
```
Comment (trishalim) [inline homepage/gcmp/lib/posts.tsx:3]: @aeplay removed meta from here

Comment (trishalim) [reply homepage/gcmp/lib/posts.tsx:3]: is that what you meant?

**Comment by aeplay** [comment]
Yes, perfect! Thank you

**Pull request merged**